api/v1beta2/kyma_types.go

```diff
@@ -74,6 +74,14 @@ type Module struct {
 	// +kubebuilder:validation:MinLength:=3
 	Channel string `json:"channel,omitempty"`
 
+	// Version is the desired version of the Module. If this changes or is set, it will be used to resolve a new
+	// ModuleTemplate based on this specific version.
+	// The Version and Channel are mutually exclusive options.
+	// The regular expression come from here: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+        // json:"-" to disable installation of specific versions until decided to roll this out
+        // see https://github.com/kyma-project/lifecycle-manager/issues/1847
```

Comment:
Can we mention these changes here on #1847?
Then later when need to support the version field, we are clear on what need to change. For example, change this Version to `json:version`,  enable skipped e2e test: https://github.com/kyma-project/lifecycle-manager/pull/1892/files#diff-d8d3cb73bc106e357ced5f42d78f3a065c63307df9a8e33268e2161d61d686f7R15, and something you think worth to mention.